Migration Assistant: Add info about managed service snapshots (#9651)
opensearch-project/documentation-website: _migration-assistant/deploying-migration-assistant/configuration-options.md

All of these authentication options apply to both source and target clusters.

> ## Bring-your-own-snapshot options

Reviewer: Line 154: "AWS Signature Version 4"

> The following configuration options customize migrating from snapshots.

Reviewer: ```suggestion
The following configuration options customize the process of migrating from snapshots.
```

### Snapshot of a managed service source

> If your source cluster is on Amazon OpenSearch Service, you need to set up an additional IAM role and pass it with the snapshot creation call, as described in the [AWS documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/managedomains-snapshots.html). The Migration Assistant can automatically manage this process. Managed Service snapshots are only compatible with SigV4 auth. The following parameter ensures that the additional IAM role is created and passed.

Reviewer: ```suggestion
If your source cluster is on Amazon OpenSearch Service, you need to set up an additional AWS Identity and Access Management (IAM) role and pass it with the snapshot creation call, as described in the [AWS documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/managedomains-snapshots.html). Migration Assistant can automatically manage this process. OpenSearch Service snapshots are only compatible with AWS Signature Version 4 authentication. The following parameter ensures that the additional IAM role is created and passed.
```

| Name  | Example | Description |
| :--- | :--- | :--- |

> | `managedServiceSourceSnapshotEnabled` | `true` | Creates the necessary roles and trust relationships to take a snapshot of a managed service source cluster. This is only compatible with SigV4 auth.|

Reviewer: ```suggestion
| `managedServiceSourceSnapshotEnabled` | `true` | Creates the necessary roles and trust relationships for taking a snapshot of an OpenSearch Service source cluster. This is only compatible with AWS Signature Version 4 authentication.|
```

> ### Bring-your-own-snapshot

Reviewer: ```suggestion
### Bring your own snapshot
```

```
{% include copy.html %}

> By default, Amazon S3 buckets automatically allow roles in the same AWS account (with the appropriate `s3:*` permissions) to access the S3 bucket, regardless of the bucket's AWS Region. If the external S3 bucket is in the same AWS account as the Migration Assistant deployment, no further AWS Identity and Access Management (IAM) configuration is required to access the bucket.

Reviewer: ```suggestion
By default, Amazon S3 buckets automatically allow roles in the same AWS account (with the appropriate `s3:*` permissions) to access the S3 bucket, regardless of the bucket's AWS Region. If the external S3 bucket is in the same AWS account as the Migration Assistant deployment, no further IAM configuration is required to access the bucket.
```
